cpp/oneapi/dal/table/common.hpp

```diff
@@ -96,6 +96,9 @@ class ONEDAL_EXPORT table {
     /// Swaps the implementation of this object and another one.
     table& operator=(table&&);
 
+    /// Destroys the table instance and releases the associated implementation.
+    ~table() = default;
```

Comment:
Should it be declared virtual? I think `homogen_table`, `heterogen_table`, etc. are derived from `table` and their destructors should be polymorphic.

Reply:
Oh, it seems that the virtual destructor adds or changes the vtable in all the classes derived from `table`, which impacts the ABI.

So, `virtual` cannot be added here now. Can you at least add a TODO-note to make it virtual later (in 2026.0)?